rasterio/_base.pyx

```diff
@@ -753,7 +752,7 @@ cpdef eval_window(object window, int height, int width):
     return (r_start, r_stop), (c_start, c_stop)
 
 
-def get_index(x, y, affine, op=math.floor, precision=6):
+def get_index(x, y, transform, op=math.floor, precision=6):
```

Comment:
@geowurster here and `get_window` below: it might be a good idea to pass `transform` through `guard_transform` since this is often passed by position rather than as a `kwarg`, and we are indexing directly into this tuple below.